Update readme.md

DarkStoorM opened the pull request:
`npm i` will now automatically build the source and clone the environment file

* applies Markdown lints

🏃‍♂️ 💨

DarkStoorM committed:
Merge postinstall tasks

Build the source after installing dependencies, there's no need to do this manually on a fresh install

DarkStoorM committed:
Remove code building step, fix typo

Source will now auto-build during the dependency install

DarkStoorM committed:
Fix indentation for code blocks

Fix for Markdown rule MD029, code blocks between the list elements require three spaces.

Refer to [this comment](https://github.com/DavidAnson/markdownlint/issues/125#issuecomment-395127149)

DarkStoorM committed:
Surround fenced code blocks with blank lines

Refer to Markdown rule MD031

DarkStoorM committed:
Revert "Merge postinstall tasks"

This reverts commit 1ab21b9bc520bb11b4bf3be7b75d60519b62f382.

DarkStoorM committed:
Merge postinstall tasks

DarkStoorM committed:
Add postinstall tasks, update dependencies

Hofled commented on package.json:
```diff
@@ -10,7 +10,7 @@
     "start": "node dist/index.js",
     "cloneEnv": "node dist/scripts/cloneEnv.js --clone",
-    "postinstall": "npm run db:migrate",
+    "postinstall": "npm run db:migrate && npm run build:src && npm run cloneEnv",
```
Please add `concurrently` to the project, and use that to run multiple npm commands instead of the npm native way. @DarkStoorM

Hofled reviewed (changes requested):
Please refer to [this](https://www.npmjs.com/package/concurrently) as a solution.

DarkStoorM committed:
Add Concurrently

Adding `--max-processes 1` argument was required to avoid running the postinstall tasks in parallel - this will not run `cloneEnv` until `npm run build:src` process is finished.

DarkStoorM commented:
🎉

dcepil reviewed (approved):
Looking good 🤸

Hofled commented:
I don't have access to a computer to confirm that it functions properly on a new environment but it indeed looks good 👍
Feel free to merge it when you can
